WordPress/WordPressDraftActionExtension/Tracks+DraftAction.swift

```diff
@@ -42,6 +42,15 @@ extension Tracks {
         trackExtensionEvent(.categoriesSelected, properties: properties as [String: AnyObject]?)
     }
 
+    public func trackExtensionPostTypeOpened() {
+        trackExtensionEvent(.postTypeOpened)
+    }
+
+    public func trackExtensionPostTypeSelected(_ postType: String) {
+        let properties = ["post_type_tags": postType]
+        trackExtensionEvent(.postTypeSelected, properties: properties as [String: AnyObject]?)
```

Comment:
Same as above.

Reply:
Had my search filter set to the ShareExtension folder only, missing this one. 🤦